pr-data.csv

```diff
@@ -6377,7 +6377,7 @@ https://github.com/soot-oss/soot,23758df0ce8327bb414d982aeb325a4f5e83ccf8,.,soot
 https://github.com/soot-oss/soot,23758df0ce8327bb414d982aeb325a4f5e83ccf8,.,soot.toolkits.graph.TestDominance.TestSimpleDiamond,ID,Accepted,https://github.com/soot-oss/soot/pull/1802,
 https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.compare.TestPackageSheet.tesConcludedLicensetCol,ID,Accepted,https://github.com/spdx/tools/pull/312,
 https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.compare.TestPackageSheet.testDeclaredLicenseCol,ID,Accepted,https://github.com/spdx/tools/pull/312,
-https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.rdfparser.license.TestLicenseInfoFactory.testParseSPDXLicenseStringMixedCase,ID,,,
+https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.rdfparser.license.TestLicenseInfoFactory.testParseSPDXLicenseStringMixedCase,ID,Unmaintained,,
```

Comment:
Please add a note about the last commit https://github.com/TestingResearchIllinois/idoft/blob/main/readme.md?plain=1#L152
Also may want to change the checker to check for this.

Last but not least, consider if https://github.com/TestingResearchIllinois/idoft/tree/main/auto_check_unmaintained can help you find all these cases.